DSharpPlus/Clients/DiscordClient.cs

```diff
@@ -799,6 +817,7 @@ private void UpdateMessage(DiscordMessage message, TransportUser author, Discord
                 message.Author = this.UpdateUser(usr, guild?.Id, guild, member);
             }
 
+            //var channel = this.InternalGetCachedThread(message.ChannelId) ?? this.InternalGetCachedChannel(message.ChannelId);
```

Comment:
what's this for?

Reply:
This was to pull from the thread cache for when a message is sent. It works either way (which is why the change was commented after I made it), but the intent seems clearer this way so I'll keep it.